data_processors/pipeline/lambdas/wes_handler.py

```diff
@@ -17,10 +17,18 @@
 
 from data_processors.pipeline.domain.workflow import WorkflowStatus, WorkflowRunEventType
 from utils import libjson, ica
+from pathlib import Path
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
+# Set engine parameters
```

Comment:
These will come from SSM parameter store. Ditto moving to `WorkflowHelper ` or `EngineParametersHelper(Helper)`.

Reply:
Yet to create these ssm parameters, wanted some feedback on the helper objects first

Reply:
Yup; engine param helper looks good now! Thanks